keyboards/xd75/keymaps/skewwhiffy/keymap.c

```diff
@@ -214,75 +254,3 @@ const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt)
       }
     return MACRO_NONE;
 };
-
-bool lightsOn = true;
-int keyPresses = 0;
-int effect = 0;
-uint32_t key_timer;
-
-void resetCounts(void) {
-  effect++;
-  keyPresses = 0;
-  key_timer = timer_read32();
-}
-
-void matrix_scan_user(void) {
```

Comment:
If you want the light effects still, you should use `layer_state_set_user` rather than this.

https://docs.qmk.fm/#/custom_quantum_functions?id=layer-change-code

Reply:
Nah: they were distracting :)

Reply:
Okay. Totally understand that. :)